app/controllers/stafftools/users_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Stafftools
+  class UsersController < StafftoolsController
+    before_action :set_user
+
+    def impersonate
+      if @user
+        impersonate_user(@user)
+        redirect_to root_path
+      else
+        flash[:error] = 'This user does not exist on Classroom for GitHub'
+        redirect_to :back
+      end
+    end
+
+    def stop_impersonating
+      stop_impersonating_user
```

Comment:
Is a gem really needed for this? It seems like we just need to set `session[:impersonated_user_id]`.